src/java.desktop/windows/classes/sun/awt/windows/WWindowPeer.java

```diff
@@ -662,6 +662,13 @@ boolean isTargetUndecorated() {
      @Override
      public native void repositionSecurityWarning();
 
+    @Override
+    public GraphicsConfiguration getAppropriateGraphicsConfiguration(
+            GraphicsConfiguration gc)
+    {
+        return gc;
+    }
+
```

Comment:
Same here, WWindowPeer is a WCanvasPeer descendant.

Reply:
Updated.